### DISCUSSION TITLE
eIDAS support

### DISCUSSION THREAD

**Discussion by fmerian** [documenso/documenso#285] [Ideas]
Source: [Product Hunt AMA session, May 2023](https://documen.so/ama/eIDAS)

**Comment by fmerian** [comment]
For the record, the current Release `0.9` will support the eIDAS levels as follows:

## Level 1 - Simple Electronic Signature (SES)

Full Compliance

## Level 2 - Advanced electronic signatures (AdES)

Partially Compliant

- We do employ actual signing using our own certificate since we did not want Documenso to be yet another PDF-Image-Editor
- Our signing algorithm differs from the eIDAS standard, making us an n "Indeterminate AdES digital signature" like DocuSign and Digitsigner for example, since we did not go exactly by the DSS - Digital Signature Service reference implementation
- Also we do not yet employ the dedicated signing hardware (HSM Hardware Security Modules) and audits required handling certificates of this level issues by QTSP (Qualified Trust Service Providers)

## Level 3 - Qualified Electronic Signature (QES)

Not compliant

Some main points why we can't offer QES yet are:
- Identification: We need to validate the signer's identity via Passport or similar via camera
- Again, no dedicated hardware and audits

**Comment by fmerian** [comment]
For the record, same question/request by Daniel '[@Dayjay](https://twitter.com/Dayjay)' on Twitter / X: [Direct link](https://twitter.com/Dayjay/status/1698983050573427177?s=20)

**Comment by homberger** [comment]
Are you aware of the project "Open eCard"?

https://github.com/ecsec/open-ecard
https://www.openecard.org/en/startpage/
https://dev.openecard.org/

**Comment by daallgeier** [comment]
Really looking forward to this support so i can sign documents with my Digital Government issued ID cards (E-Residency 🇪🇪, ID-Austria 🇦🇹) rather than the "please take a picture of yourself and your passport" system.

**Comment by masoudtahmasebi** [comment]
Hi thank you for the great system.
can I ask if there is timeline that Level 2 - Advanced electronic signatures (AdES) is going to be available?

**Comment by ivannovazzi** [comment]
For anyone who needs eIDAS compliance *right now* without waiting for it to land in Documenso, [Gatherly](https://news.ycombinator.com/item?id=47085201) ships qualified eIDAS signatures out of the box. It bundles e-signature + document collection into a single Secure Link — the client signs and uploads everything (ID docs, supporting files, etc.) in one portal without needing an account.

The eIDAS Level 2/3 requirement comes up a lot for law firms and accounting firms who need tamper-evident, legally-standing documents for EU clients. Documenso covering Level 1 is a great step — just flagging the alternative for anyone blocked on the higher levels today.
